libs/sdk-common/mock_service_worker.js

```diff
@@ -0,0 +1,93 @@
+let db = null;
+
+async function getDb() {
+    if (db) {
+        return db;
+    } else {
+        return await new Promise((rs, rj) => {
+            const req = indexedDB.open("http_mock", 1);
+            req.onsuccess = (event) => rs(event.target.result);
+            req.onerror = rj;
+            req.onupgradeneeded = (event) => {
+                db = event.target.result;
+                db.createObjectStore("mocks", { keyPath: "nonce" });
+            };
+        })
+    }
+}
```

Comment:
When running the tests in non-headless mode, I'm getting an error after running the tests for the first time:

On Chrome:
```
DataError: Failed to execute 'get' on 'IDBObjectStore': No key or key range specified.
```
On Firefox:
```
DataError: No valid key or key range specified.
```

If I delete browser caches, cookies, etc, then it works once again.

I assume the culprit is failing to the get the db here.

Reply:
I can't reproduce this on chrome/firefox, but will keep testing

Reply:
Thanks. On my machine the behavior is the same regardless of the browser. I spent some time trying to fix it but didn't manage to. IMO it isn't a blocker as it should be caused by this mock and it doesn't affect headless runs.